install.sh

```diff
@@ -101,13 +101,9 @@ function install_snowboy()
     if install_dependencies
     then
         root_dir=$(pwd)
-        git clone https://github.com/Kitt-AI/snowboy.git
-        cd snowboy/swig/Python3
-        make -j4
+        sudo pip3 install git+https://github.com/Kitt-AI/snowboy.git
         if [ -f _snowboydetect.so ]; then
```

Comment:
Snowboy now is no longer built inside susi_linux folder, this `if` line will fail.

Reply:
I am not deleting the snowboy folder here @hongquan.
Will do that in the other PR

Reply:
updated it